test/e2e/certificates_test.go

```diff
@@ -435,6 +452,202 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("dns-01 challenge using explicit credentials on ibm cloud CIS", func() {
+		It("should obtain a valid LetsEncrypt certificate", func() {
+			cisCRN, isCisCRN := os.LookupEnv(cisCRNEnvironmentVar)
+			if targetPlatform, ok := os.LookupEnv(targetPlatformEnvironmentVar); ok && targetPlatform == "ibmcloud" {
+				if !isCisCRN || cisCRN == "" {
+					Fail("cisCRN is required for IBM Cloud platform")
+				}
+			} else {
+				Skip("skipping as the cluster does not use IBM Cloud CIS")
+			}
+
+			By("creating a test namespace")
+			ns, err := loader.CreateTestingNS("e2e-acme-explicit-dns01")
+			Expect(err).NotTo(HaveOccurred())
+			defer loader.DeleteTestingNS(ns.Name)
+
+			By("creating new certificate ClusterIssuer with IBM Cloud CIS webhook solver")
+			clusterIssuerName := "letsencrypt-dns01-explicit"
+			By("delete existing ClusterIssuer with the same name if it exists")
+			_ = certmanagerClient.CertmanagerV1().ClusterIssuers().Delete(ctx, clusterIssuerName, metav1.DeleteOptions{})
+
+			clusterIssuer := &certmanagerv1.ClusterIssuer{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      clusterIssuerName,
+					Namespace: ns.Name,
+				},
+				Spec: certmanagerv1.IssuerSpec{
+					IssuerConfig: certmanagerv1.IssuerConfig{
+						ACME: &v1.ACMEIssuer{
+							Server: "https://acme-v02.api.letsencrypt.org/directory",
```

Comment:
```suggestion
							Server: "https://acme-staging-v02.api.letsencrypt.org/directory",
```
For e2e ci, it would be better to use [staging](https://letsencrypt.org/docs/staging-environment/) endpoint.

Reply:
Will do